README.md

```diff
@@ -154,16 +231,27 @@ To avoid accidentally exceeding the rate limit on [Google's Places API][google-p
       You can assign yourself other roles in the same way as creating the
       organizer role. If however you assign yourself a student role AND another
       role, that may lead to unexpected behavior in the app. In that case,
-      remove the student role. 
-    * Refresh the browser and you should see links for organizers. 
-- Once you are an `organizer`, you can add a season and switch between season's phases at
+      remove the student role.
+
+Once you are an `organizer`, you can add a season and switch between season's phases at
 http://localhost:3000/orga/seasons in your browser.
-- While in development, you are also able to impersonate other users to easily test the system
+
+While in development, you are also able to impersonate other users to easily test the system
 as someone else. Go to http://localhost:3000/users while logged in to do that.
-- You are good to go now. Happy coding!
+
+---
+
+You are good to go now. Happy coding!
+
+--
 
 ## Testing
 
+Please write relevant tests as you code, and test locally before opening a pull request. 
+Thank you! 
+
```

Comment:
I'd vote to tell here only how to run the tests locally. Info about writing tests should either be more helpful, or in the contribution guide, or not here. Wdyt?